spec/30-processing-model.md

```diff
@@ -0,0 +1,66 @@
+# Processing Model
+
+This section defines the behaviour of a tracing system when a new request is received by a component. As stated above tracing systems can decide for each individual request whether they want to _forward_ a trace or _participate_ in a trace. The processing model for both scenarios will be described individualy and adds a seperate processing model when a tracing system is the first system to start a trace. 
+
+## Processing Model in Forwarding Mode
+
+In forwarding mode a tracing system decides to forward tracing information but to not modify the trace-context headers. The request is then considered as not sampled. 
+
+1. Tracing system receives a `traceparent` header and may receive `tracecontext`. 
+2. Tracing system tries to parse the version of the `traceparent` header.
+   1. If the version prefix cannot be parsed (it's not 2 hex characters followed by
+   dash (`-`)), implementation should restart the trace.
+*NOTE* : If a system is in forwarding only mode. It cannot start a trace. 
+   2. If the version is higher than supported by the tracing system, does no further validations. 
+3. When the version number is supported by the tracing system it validates `trace-id` and `parent-id`
+  1. If `trace-id` and `parent-id` are valid, the tracing system adds the entire `trace-context` to all outgoing request orginating from the current incoming request. 
+  2. If either `trace-id` or `parent-id` are invalid, the tracing systems does not add the trace context to any outgoing requests.
+*NOTE* : We never explictly defined this, but I think this is what should happen. 
+
+## Processing Model in Participation Mode
+
+In participation mode the tracing system decides to forward AND modofiy trace-context headers. The request is then consideres as sampled.
+
+1. Tracing system receives a `traceparent` header and may receive `tracecontext`. 
```

Comment:
please make sub-lists render correctly:

![image](https://user-images.githubusercontent.com/9950081/49888272-651bc880-fdf3-11e8-881b-7494fcea4f9b.png)

Reply:
this should be fixed.